snap/info.go

```diff
@@ -526,11 +530,17 @@ func (s *Info) UserCommonDataDir(home string, opts *dirs.SnapDirOptions) string
 	return UserCommonDataDir(home, s.InstanceName(), opts)
 }
 
+// NewSnapUserHome returns the new upper-case snap directory in the user home.
+func (s *Info) NewSnapUserHome(home string) string {
```

Comment:
we should find some consistent naming for this, something like:

UserExposedHomeDir
UserExposedDataDir